examples/overview.ts

```diff
@@ -45,7 +45,9 @@ async function main() {
   // example: TRANSACTION_PAYMENT_SUBMIT
 
   // example: ALGOD_FETCH_ACCOUNT_INFO
-  const acctInfo = await algodClient.accountInformation(acct.addr).do();
+  const acctInfo = await algodClient
+    .accountInformation(acct.addr.toString())
```

Comment:
I think so, yes. I was planning to defer this decision to the next PR, which modifies the request types already, but perhaps it would be better to add this now